src/main/target/KAKUTEF4/target.h

```diff
@@ -78,6 +78,7 @@
 #define USE_MAX7456
 #define MAX7456_SPI_INSTANCE    SPI3
 #define MAX7456_SPI_CS_PIN      PB14
+#define MAX7456_SPI_CS_OPU               // Use explicit output pull-up
```

Comment:
Do we really need to introduce this new define, or can we solve this in a way that will translate directly into generic targets in `targetConfiguration()`?

Reply:
I added a code in KAKUTEF4 `config.c` to remove `MAX7456_SPI_CS_PIN` from IPU array and add it to OPU array, for legacy targets.

For generic target, all IPU and OPU will be added by `resource` commands.
E.g., for KAKUTEF4:
```
resource spi_preinit_opu B14
```